<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsWindow</class>
 <widget class="QDialog" name="settingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>data/icon.ico</normaloff>data/icon.ico</iconset>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">color:black;</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Press esc or exit to cancel</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonArray">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>100</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Discard|QDialogButtonBox::RestoreDefaults|QDialogButtonBox::Save</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <item row="1" column="1">
     <widget class="QLineEdit" name="labelListPath">
      <property name="text">
       <string>data/faceLabels.csv</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Label Config</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="labelConfigPath">
      <property name="text">
       <string>data/labelConfig.csv</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Image folder</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="imageDir">
      <property name="styleSheet">
       <string notr="true">&lt;font color=red&gt;</string>
      </property>
      <property name="text">
       <string>data/Images</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>label output file</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
